src/components/SettingsPage/SyncCard.tsx

```diff
@@ -152,59 +155,77 @@ export const SyncCard = () => {
-          container
-          alignItems="center"
-          direction={'row'}
-          justifyContent="center"
-          style={{ marginBottom: '2px' }}
-        >
-          <Grid item lg={10} data-testid={`PollingLinearGauge-entity-sync`}>
-            <PollingLinearGauge
-              done={false}
-              poll={updateEntitySyncProgress}
-              value={syncTaskProgressPercentage}
+    <>
+      {cortexSyncError !== undefined && (
+        <WarningPanel severity="error" title={cortexSyncError}>
+          There is already a sync in progress. Try again in 10 minutes.
+        </WarningPanel>
+      )}
+      <InfoCard
+        title="Sync entities"
+        action={
+          cortexSyncError === undefined && (
+            <SyncButton
+              isSyncing={
+                syncTaskProgressPercentage !== null || isSubmittingTask
+              }
+              submitSyncTask={async () => {
+                try {
+                  await submitEntitySync();
+                  setCortexSyncError(undefined);
+                } catch (e: any) {
+                  setCortexSyncError(e.message);
+                }
+              }}
             />
+          )
+        }
+      >
+        {syncTaskProgressPercentage !== null && (
+          <Grid
+            container
+            alignItems="center"
+            direction={'row'}
+            justifyContent="center"
+            style={{ marginBottom: '2px' }}
+          >
+            <Grid item lg={10} data-testid={`PollingLinearGauge-entity-sync`}>
+              <PollingLinearGauge
+                done={false}
+                poll={updateEntitySyncProgress}
+                value={syncTaskProgressPercentage}
+              />
+            </Grid>
+            <Grid item lg={2}>
+              <CancelSyncButton cancelSync={cancelEntitySync} />
+            </Grid>
           </Grid>
-          <Grid item lg={2}>
-            <CancelSyncButton cancelSync={cancelEntitySync} />
-          </Grid>
-        </Grid>
-      )}
-      {lastSyncedTime !== null ? (
-        <Typography>
-          <i>
-            Last synced on{' '}
-            {moment
-              .utc(lastSyncedTime)
-              .local()
-              .format('MMM Do YYYY, h:mm:ss a')}
-          </i>
-        </Typography>
-      ) : (
+        )}
+        {lastSyncedTime !== null ? (
+          <Typography>
+            <i>
+              Last synced on{' '}
+              {moment
+                .utc(lastSyncedTime)
+                .local()
+                .format('MMM Do YYYY, h:mm:ss a')}
+            </i>
+          </Typography>
+        ) : (
+          <Typography>
+            <i>Entities have never been synced before.</i>
+          </Typography>
+        )}
         <Typography>
-          <i>Entities have never been synced before.</i>
+          Manually sync your Backstage entities with Cortex.
+          <br />
+          You can also set this up to automatically sync with our&nbsp;
```

Comment:
nit: while you're here, `&nbsp;` and a space are slightly different for esoteric HTML reasons
```suggestion
          You can also set this up to automatically sync with our{' '}
```